cilium-dbg/cmd/bgp_route_get.go

```diff
@@ -172,15 +182,19 @@ func parseBGPPeerAddr(args []string) (string, error) {
 	return addr.String(), nil
 }
 
-func printBGPRoutesTable(routes []*models.BgpRoute) {
-	// sort first by ASN and then by prefix
+func printBGPRoutesTable(routes []*models.BgpRoute, printPeer bool) {
+	// sort first by ASN, then by neighbor, then by prefix
 	sort.Slice(routes, func(i, j int) bool {
-		return routes[i].RouterAsn < routes[j].RouterAsn || routes[i].Prefix < routes[j].Prefix
+		return routes[i].RouterAsn < routes[j].RouterAsn || routes[i].Neighbor < routes[j].Neighbor || routes[i].Prefix < routes[j].Prefix
 	})
 
 	// get new tab writer with predefined defaults
 	w := NewTabWriter()
-	fmt.Fprintln(w, "VRouter\tPrefix\tNextHop\tAge\tAttrs")
+	if printPeer {
+		fmt.Fprintln(w, "VRouter\tPeer\tPrefix\tNextHop\tAge\tAttrs")
+	} else {
+		fmt.Fprintln(w, "VRouter\tPrefix\tNextHop\tAge\tAttrs")
```

Comment:
Why do we need to omit `Peer` from the output? 

If I understand correctly, user can issue command to get routes without specifying peer. But in the output, we can still provide peer information, right ?

Reply:
The main reason is that for `available` routes (`loc-rib`), there is no peer information, so it makes no sense to print the `Peer` column, as it would be always empty for each route.

But yeah, for `advertised` routes (`adj-rib-out`), when a peer is specified, it is just an optimization, to fit more data within the terminal width.